.evergreen/config.yml

```diff
@@ -293,6 +293,54 @@ functions:
           export NODE_LTS_NAME='${NODE_LTS_NAME}'
 
           bash ${PROJECT_DIRECTORY}/.evergreen/run-atlas-tests.sh
+  run socks5 tests:
+    - command: shell.exec
+      type: test
+      params:
+        silent: true
+        working_dir: src
+        script: |
+          ${PREPARE_SHELL}
+          cat <<EOT > prepare_client_encryption.sh
+          export CLIENT_ENCRYPTION=${CLIENT_ENCRYPTION}
+          export CSFLE_KMS_PROVIDERS='${CSFLE_KMS_PROVIDERS}'
+          export AWS_ACCESS_KEY_ID="${AWS_ACCESS_KEY_ID}"
+          export AWS_SECRET_ACCESS_KEY="${AWS_SECRET_ACCESS_KEY}"
+          export CSFLE_GIT_REF="${CSFLE_GIT_REF}"
+          export CDRIVER_GIT_REF="${CDRIVER_GIT_REF}"
+          EOT
+    - command: shell.exec
+      type: test
+      params:
+        working_dir: src
+        script: >
```

Comment:
TIL about this character "folding newlines", so when parsed this string will won't have the empty lines? 😎